db/schema.rb

```diff
@@ -328,7 +320,7 @@
     t.string   "last_sign_in_ip"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "organisation",                             null: false
```

Comment:
This was null: false as organisation is effectively the primary key. Any reason to change?